qemu/tests/nic_opt.py

```diff
@@ -128,7 +128,7 @@ def test_ping():
 
     check_type = params.get("check_type")
     smp_value = params.get_numeric("smp") or params.get_numeric("vcpu_maxcpus")
-    if cpu.online_count() < 2 * smp_value:
+    if cpu.online_count() < 0.25 * smp_value:
```

Comment:
I think we should not test cpu overcommit scenario, when `cpu.online_count() < 2 * smp_value`, modify `params['smp']` and `params['vcpu_maxcpus']`, ensure they are smaller than `cpu.online_count`.

Reply:
Or remove this variant on our test plan.

Reply:
Hi @PaulYuuu  I want to change this variant to be less than the number of queues, equal to the number of queues, and large than the number of queues. Because we have such a product bug. When the smp is equal to the number of queues and large than the number of queues, qemu outputs some error info. What do you think of this test plan ?

Reply:
Sure, we can test it, please note that this may cause the guest to start abnormally, if the CPU overcommit ratio is too high.

Reply:
Ok, thanks, I lowered the number of CPUs. Please help me review it again, thanks a lot.